mods/action.js

```diff
@@ -10,75 +10,75 @@ var Template = require('../lib/reqTemplate');
  * Error translation
  */
 var errDefs = {
-    '400': { status: 400, type: 'invalid_request' },
-    '401': { status: 401, type: 'unauthorized' },
-    '403': { status: 403, type: 'access_denied#edit' },
-    '409': { status: 409, type: 'conflict' },
-    '413': { status: 413, type: 'too_large' },
-    '429': { status: 429, type: 'rate_exceeded' },
-    '500': { status: 500, type: 'server_error' },
-    '501': { status: 501, type: 'not_supported' }
+    400: { status: 400, type: 'invalid_request'},
```

Comment:
Why? If there is a space after the opening brace, I'd expect there to be one at the end as well (symmetry, OCD and all) :)